src/components/QuoteBubble/QuoteBubble.js

```diff
@@ -1,16 +1,24 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import { speechBubbleVariant } from '../../types';
+import extractRoleFromPath from '../../utils/extractRoleFromPath';
+import extractActionFromPath from '../../utils/extractActionFromPath';
+import extractQuoteIDFromPath from '../../utils/extractQuoteIDFromPath';
+import { getQuote } from '../../utils/quotesService';
+import { match, speechBubbleVariant } from '../../types';
 import { SPEECH } from '../../constants/speechBubbleVariant';
 import CopyButton from './CopyButton';
 import Wrapper from './Wrapper';
 import Quote from './Quote';
 import BubbleTail from './BubbleTail';
 
-const DEFAULT_QUOTE = "Daily in 5 minutes and I'm still not sure what to say...";
+function QuoteBubble({ variant = SPEECH, noCopyToClipboard = false, match: { url } }) {
```

Comment:
Not a big fan of such nested destructuring. I think `const { url } = match` followed by new line would be a bit more cleaner.